### PR TITLE
Lesson 1.1

### PR DESCRIPTION
**Pull request by KgSolomon**
React App and list created.

### CODE REVIEW

**Review by kwilson05 — COMMENTED** [review]
Looking good! 💯

File: src/App.js [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Todo List</h1>
+      <ul>
+        {todoList.map(function (item) {
+          return (<li key={item.id}>{item.title}</li>)
+        })}
+      </ul>
```
Comment (kwilson05) [inline src/App.js:27]: Nice job here! Not a blocker

Maybe try an implicit return and use an arrow function next time. It makes the code look a bit cleaner. Something like this:
`todoList.map(item => (<li key={item.id}>{item.title}</li>))`

Let me know if this doesn't make sense. I can explain further

**Review by kwilson05 — CHANGES_REQUESTED** [review]
The only blocker here is you didn't submit your package.json file. The file holds all your dependencies needed for the project, like React.

Nvm i see what you did! 👍

**Review by kwilson05 — APPROVED** [review]
Looking good! 💯

File: src/App.js (hunk 1 above)
Comment (KgSolomon) [reply src/App.js:27]: Thanks! I'll be using arrow functions next time.

**Pull request merged**